src/pages/ProfilePage.tsx

```diff
@@ -90,10 +90,13 @@ function ProfilePage({personalDetails, personalDetailsMetadata, route, session,
     const fallbackIcon = details?.fallbackIcon ?? '';
     const login = details?.login ?? '';
     const timezone = details?.timezone;
+    const reportRecipient = personalDetails?.[accountID];
+    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+    const isParticipantValidated = reportRecipient?.validated || false;
```

Comment:
```suggestion
    const isParticipantValidated = reportRecipient?.validated ?? false;
```

Why did we add `// eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing` instead of using `??` as the list error suggested to? the lint checks were added for a reason and are not meant to be disabled without a good reason. Is there a good reason in this case to disable the check?

Reply:
Thanks for fixing! @mollfpr @GandalfGwaihir always avoid disabling rules unless there is a good reason for it :)